pepper-apis/src/main/java/org/broadinstitute/ddp/customexport/export/CustomExportCoordinator.java

```diff
@@ -111,12 +113,14 @@ private void sendNotificationEmail(boolean isFailure, boolean isSkip) {
         } else {
             subject = cfg.getString(CustomExportConfigFile.EMAIL_SUCCESS_SUBJECT);
             templateId = cfg.getString(CustomExportConfigFile.EMAIL_SUCCESS_TEMPLATE_ID);
-            String fileUrl = "https://console.cloud.google.com/storage/browser/" + exportCfg.getString(CustomExportConfigFile.BUCKET_NAME)
-                    + "/" + exportCfg.getString(CustomExportConfigFile.FILE_PATH);
+            String fileUrl = "https://console.cloud.google.com/storage/browser/" + mainCfg.getString(CustomExportConfigFile.BUCKET_NAME)
+                    + "/" + mainCfg.getString(CustomExportConfigFile.FILE_PATH);
```

Comment:
`mainCfg` is not the `CustomExportConfigFile`, is it?

Reply:
Nope.  It comes from here: ` public static void register(Scheduler scheduler, Config cfg) throws SchedulerException {`

Reply:
Wait let me check this one more time.  I swear I did this right but now I'm worried

Reply:
Nope; wrote a totally different bug.  Clearly I'm a little off today!

Reply:
Looks much better!